### ISSUE TITLE
Farming Simulator 22

### ISSUE THREAD

**Issue by doc3d** [Boomslangnz/FFBArcadePlugin#86]
i LOVE the idea of your plugin. This must be helpful to a whole bunch of people. I was wondering if there is any way to make FS22 compatible? I dont know if there is a config per game, or if there is a "universal" setup that can be used. Either way, FS22 support would mean a lot to the people playing FS22, if at all possible to make such requests.

**Comment by DFox-84** [comment]
I agree with what is written above , I am looking for a plug-in for fs25 and other simulators
